src/components/evaluation/EvaluationActions.vue

```diff
@@ -271,31 +263,52 @@ export default {
       }
       return fields
     },
-    handleError(error) {
-      this.showErrorDialog(this.$_.get(error, 'response.data.message', 'An unknown error occurred.'))
-      this.refreshEvaluations().finally(() => {
-        this.setDisableControls(false)
-        this.applyingAction = null
-        this.isApplying = false
-        this.isLoading = false
-      })
-
-    },
     isInvalidAction(action) {
       const uniqueStatuses = this.$_.uniq(this.evaluations
                   .filter(e => this.selectedEvaluationIds.includes(e.id))
                   .map(e => e.status))
-
       return (uniqueStatuses.length === 1 && uniqueStatuses[0] === action.status)
     },
-    refreshEvaluations() {
-      const selectedCourseNumbers = this.$_.uniq(this.evaluations
-                  .filter(e => this.selectedEvaluationIds.includes(e.id))
-                  .map(e => e.courseNumber))
-      if (selectedCourseNumbers.length === 1) {
-        return this.refreshSection({sectionId: selectedCourseNumbers[0], termId: this.selectedTermId})
-      }
-      return this.refreshAll()
```

Comment:
The method above is invoked only in the _handle-error_ logic, so that logic was moved to line 296 below.